python/mlc_llm/serve/engine.py

```diff
@@ -329,7 +330,6 @@ def create(  # pylint: disable=too-many-arguments,too-many-locals
         n: int = 1,
         seed: Optional[int] = None,
         stop: Optional[Union[str, List[str]]] = None,
-        stream: Literal[False] = False,
```

Comment:
Keep it in this way, as when not specified the default value is `False`.

Reference: https://github.com/openai/openai-python/blob/main/src/openai/resources/completions.py#L40-L56

Reply:
This is one of the two overload signatures (instead of the function definition itself) where the stream value is false. The other signature (stream = True) does not provide a default value and just annotate it as `Literal[True]`, so for this one I feel we can be consistent with that.

Reply:
>  The other signature (stream = True) does not provide a default value and just annotate it as `Literal[True]`

Yes this is the right way, because `stream=True` only when it's explicitly given. Otherwise, the typing system cannot figure out things. For example the mypy fails here https://ci.mlc.ai/blue/organizations/jenkins/mlc-llm/detail/PR-2515/2/pipeline.